decentraland/explorer-desktop-launcher
# fix: add entitlements info.plist with NSMicrophoneUsageDescription key

## What does this PR change?

On MacOS desktop version we have issue with voice chat not registering any input.
That is due not properly setting permission for the os.

## How to test the changes?

Check if Decentraland app appears in System Preferences/Security & Privacy/Privacy/Microphone
Or the app requests for the permission during start.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
